Modules/LearningSequence/classes/Setup/class.ilLearningSequenceSetupAgent.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/* Copyright (c) 2020 Nils Haagen <nils.haagen@concepts-and-training.de> Extended GPL, see docs/LICENSE */
+
+use ILIAS\Setup;
+use ILIAS\Refinery;
+use ILIAS\UI;
+
+class ilLearningSequenceSetupAgent implements Setup\Agent
+{
+    /**
+     * @inheritdoc
+     */
+    public function hasConfig() : bool
+    {
+        return false;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function getConfigInput(Setup\Config $config = null) : UI\Component\Input\Field\Input
+    {
+        throw new \LogicException("Agent has no config.");
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function getArrayToConfigTransformation() : Refinery\Transformation
+    {
+        throw new \LogicException("Agent has no config.");
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function getInstallObjective(Setup\Config $config = null) : Setup\Objective
+    {
+        $dir_objective = new ilFileSystemComponentDataDirectoryCreatedObjective(
+            'LSO',
```

Comment:
Hi @nhaagen,

I wonder where this magic value derives from. Is there any constant we could use here instead?

Best regards!